tests/test_time_utils.py

```diff
@@ -53,6 +54,17 @@ def test_max_time(self):
         value_max = time_utils.astropy_to_nsec(time_utils.MAX_TIME)
         self.assertEqual(value, value_max)
 
+        # Check that we do not warn inside our code for UTC in the future
+        with self.assertWarns(Warning):
+            time = Time("2101-01-01T00:00:00", format="isot", scale="utc")
```

Comment:
I'm not sure I can parse that comment, the code seems to be testing that astropy makes warning here, but there is nothing for "our code"?

Reply:
I'm mostly hiding the warnings from pytest. If you'd rather I can put the warnings context manager and simple warnings filter in. Warning here at least tells us that there is a warning to hide later on.

Reply:
No, I 'm happy with what this code does, it's just comment that confuses me. When I read it initially I understood "our code" as the code in daf_butler package but now I read it again and I think you mean next two lines by "our code".

Reply:
Also there is a chance that in 100 years or less astropy will stop making warnings about 2101 and this unit test will break 🙂

Reply:
I fixed it properly.